lua/terrortown/menus/score/info.lua

```diff
@@ -29,11 +29,16 @@ local function MakePlayerRoleTooltip(parent, width, ply)
 		plyRoleBox.GetTitle = function()
 			return tostring(i) .. ". " .. TryT(roleData.name) .. " (" .. TryT(plyRole.team) .. ")"
 		end
-
+		local length = fastutf8.len(plyRoleBox.GetTitle())
+		if length > shortest then
```

Comment:
should it really be shortest then? not longest?

Also probably somehing like `lengthLongestLine`